File: pset1/mariomore/mario.c
//libraries
#include<cs50.h>
#include<stdio.h>

int get_input();


//main function
int main(void)
{
    int i = get_input();

    for (int j = i - 1; j >= 0; j--)
    {

        // print spaces
        for (int a = 0; a < j; a++)
        {
            printf(" ");
        }

        //print #
        for (int k = i; k > j; k--)
        {
            printf("#");
        }

        printf("  ");

        //print #
        for (int k = i; k > j; k--)
        {
            printf("#");
        }
        printf("\n");
    }
}


//prompt user for input
int get_input()
{
    int a;

    do
    {
        a = get_int("Height: ");
    }
    while (a >= 9 || a <= 0);

    return a;
}

File: pset6/mario/more/mario.py
from cs50 import get_int

height = 0
while height < 1 or height > 8:
    height = get_int("Height: ")
a = 1
b = height - 1
for i in range(height):
    print(" " * b + "#" * a + "  " + "#" * a)
    a += 1
    b -= 1